src/index.html

```diff
@@ -17,5 +17,8 @@
 
   <body>
     <header class="header">Header</header>
```

Comment:
Can you separate this on multiple lines? The bot is a bit fragile here expecting exactly the code that is given [here in the course](https://ooloo.io/project/github-flow/task-2-implementation).

```
  <header class="header">
    Header
  </header>
```